packages/suite-data/files/translations/af.json

```diff
@@ -26,5 +26,6 @@
   "FW_CAPABILITY_NO_CAPABILITY": "Nie ondersteun nie",
   "FW_CAPABILITY_NO_SUPPORT": "Nie ondersteun nie",
   "FW_CAPABILITY_UPDATE_REQUIRED": "Vereis opdatering",
+  "TR_COINS": "Kryptoměny",
```

Comment:
Czech word in `af` language probably by mistake. Not relevant for us now, `af` is not available in suite anyway...